src/components/tools/globalhotkeys.component.tsx

```diff
@@ -0,0 +1,25 @@
+import { GlobalHotKeys } from "react-hotkeys";
+import React from "react";
+import { useDispatch } from "react-redux";
+import { ActionCreators as UndoActionCreators } from "redux-undo";
+
+export function CustomGlobalHotKeys(): JSX.Element {
+  const dispatch = useDispatch();
+
+  enum KeyMapActions {
+    UNDO,
+    REDO,
+  }
+
+  const keyMap: Record<KeyMapActions, string> = {
+    [KeyMapActions.UNDO]: "ctrl+z",
```

Comment:
A czy ty testowałeś u siebie te handlery  ? Ponieważ u mnie nic się nie działo kiedy na nie klikałem

Reply:
Tak, niby wszystko działało. Sprawdzę to jeszcze w innych przeglądarkach :)

Reply:
Chyba wiem o co chodzi. Prawdopodbnie próbowałeś wywołać skrót mając zaznaczoną jakąś komórkę. W takiej sytuacji ten skróty są defaultowo używane przez input. Żeby wywołać naszą akcję undo nie wolno mieć zaznaczonej żadnej komórki. Mógłbym to oczywiście zmienić, ale myślę, że nie ma sensu, skoro i tak forma w jakiej będziemy ustawiać zmiany się zmieni.